utoipa-gen/Cargo.toml

```diff
@@ -27,7 +27,7 @@ utoipa = { path = "../utoipa", features = ["debug", "uuid"], default-features =
 serde_json = "1"
 serde = "1"
 actix-web = { version = "4", features = ["macros"], default-features = false }
-axum = { version = "0.7", default-features = false }
+axum = { version = "0.7", default-features = false, features = ["json", "query"] }
```

Comment:
Doctests failed to compile without these

Reply:
Nice, it might be that there is some upstream changes that hasn't been updated here previously.